Go/1-simple/1-simple.go

```diff
@@ -0,0 +1,17 @@
+package main
```

Comment:
In my opinion, it will be better to create an `examples/` folder and store all of the examples there instead of creating a folder for each example.